BaikalDataGenerator/Source/render.h

```diff
@@ -159,12 +163,18 @@ void Render::GenerateDataset(const TCamStatesRange<CameraInfo, Args1 ...>& cam_s
         THROW_EX("spp should be positive");
     }
 
-    SaveMetadata(output_dir);
+    SaveMetadata(output_dir,
+                 cam_states.begin()->index,
+                 (cam_states.end() - 1)->index,
```

Comment:
`end() - 1` will not work with `std::list` and `std::forward_list`.

Reply:
And GetSplitByIdx will now work with map and set...